bundler/spec/runtime/with_unbundled_env_spec.rb

```diff
@@ -72,32 +72,22 @@ def run_bundler_script(env, script)
       bundle_exec_ruby bundled_app("source.rb")
       expect(out).to eq original
     end
-  end
-
-  shared_examples_for "an unbundling helper" do
-    it "should delete BUNDLE_PATH" do
-      create_file("source.rb", <<-RUBY)
-        print #{modified_env}.has_key?('BUNDLE_PATH')
-      RUBY
-      ENV["BUNDLE_PATH"] = "./foo"
-      bundle_exec_ruby bundled_app("source.rb")
-      expect(last_command.stdboth).to include "false"
-    end
 
-    it "should remove '-rbundler/setup' from RUBYOPT" do
+    it "should remove 'bundler/setup' from RUBYOPT" do
       create_file("source.rb", <<-RUBY)
-        print #{modified_env}['RUBYOPT']
+        print Bundler.original_env['RUBYOPT']
```

Comment:
the print statement here was dynamic before because it was in the shared example; now it's just testing specifically for `original_env` and is no longer in the shared example.